helm/charts/scheduled-volume-snapshotter/values.yaml

```diff
@@ -1,7 +1,7 @@
 image:
   repository: ryaneorth/scheduled-volume-snapshotter
   tag: latest
-  pullPolicy: IfNotPresent
+  pullPolicy: Always
```

Comment:
could you provide more details about the reasoning behind this?

Reply:
Sure, without this policy, during the rollout of the snapshot scheduler to a new version, the snapshot scheduler will fail it.

Reply:
Actually, I think I will need to generate a new image anyway, so I will revert this change. Good catch!

Reply:
Yeah, that is what I imagined haha since there were code changes it is better to have a new image